hypothesis-python/docs/supported.rst

```diff
@@ -25,10 +25,10 @@ changes in patch releases.
 Python versions
 ---------------
 
-Hypothesis is supported and tested on CPython 2.7 and CPython 3.5+, i.e.
+Hypothesis is supported and tested on CPython 3.5+, i.e.
 `all versions of CPython with upstream support <https://devguide.python.org/#status-of-python-branches>`_,
 
-Hypothesis also supports the latest PyPy for both Python 2 (until 2020) and Python 3.
+Hypothesis also supports the latest PyPy for Python 3.5 and 3.6.
 Hypothesis does not currently work on Jython,
```

Comment:
We should drop this: Jython doesn't support a modern version of Python 3 and is unlikely ever to, so we are unliekly to ever support it.